helix-front/pom.xml

```diff
@@ -58,7 +58,7 @@
       <plugin>
         <groupId>com.github.eirslett</groupId>
         <artifactId>frontend-maven-plugin</artifactId>
-        <version>0.0.22</version>
+        <version>1.12.1</version>
```

Comment:
use latest version to ensure that we have support for `install-node-and-yarn` goal shown below

Reply:
[latest version](https://github.com/eirslett/frontend-maven-plugin/tags) source